draft-sharabayko-mops-srt.md

```diff
@@ -1962,10 +1962,19 @@ have no chance to be delivered in time, and allows the receiver to skip missing
 that have not been delivered in time. The timeout of dropping a packet is based on the 
 TSBPD mechanism ({{tsbpd}}).
 
-In the SRT, when Too-Late Packet Drop is enabled, and a packet timestamp is older than 
-125% of the SRT latency, it is considered too late to be delivered and may be dropped
-by the sender. However, the sender keeps packets for at least 1 second in case the
-SRT latency is not enough for a large RTT (that is, if 125% of the SRT latency is less 
+When TLPKTDROP is enabled, a packet timestamp is older than TLPKTDROP_THRESHOLD,
```

Comment:
```suggestion
With TLPKTDROP is enabled, when a packet timestamp is older than TLPKTDROP_THRESHOLD,
```

Reply:
You would either say "When TLPKTDROP is enabled ..." or "With TLPKTDROP enabled ..."

Both forms are correct.